### PR TITLE
5714 - Fix on Sink Page Tab Alignment

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will fix the tab focus indicator in sink page tab alignment.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5714
Closes https://github.com/infor-design/enterprise/issues/5837

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/
- Set focus on `Notes` tab

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - `[Searchfield]` Save input value when searchfield collapses but is not cleared via button click or key. ([#5792](https://github.com/infor-design/enterprise/issues/5792))
 - `[Tabs-Vertical]` Fixed on Tabs Vertical Aria and Roles. ([#5712](https://github.com/infor-design/enterprise/issues/5712))
 - `[Toolbar Searchfield]` Fixed the height the collapse button on a smaller viewport (`766px` and below). ([#5791](https://github.com/infor-design/enterprise/issues/5791))
+- `[Tabs]` Fix focus indicator in Sink Page. ([#5714](https://github.com/infor-design/enterprise/issues/5714))
```
Comment (yohannahbautista) [inline docs/CHANGELOG.md:14]: I think this should be above Tabs-Vertical

**Review by yohannahbautista — CHANGES_REQUESTED** [review]

**Review by tmcconechy — APPROVED** [review]
Looks good. And i think this also fixed https://github.com/infor-design/enterprise/issues/5837
So i added it but please also confirm.

**Comment by tjamesallen15** [comment]
> Looks good. And i think this also fixed #5837 So i added it but please also confirm.

Yep. I checked the #5837 and it was also fixed.

**Review by yohannahbautista — APPROVED** [review]
Looks good!

**Pull request merged**